### PR TITLE
feat: SSG Support

### PR DESCRIPTION
**Pull request by mohamadrezahedayati**

### CODE REVIEW

**Comment by mHooshdar** [comment]
Thanks a lot for adding SSG support ✌🏻

File: README.md [hunk 1]
```diff
@@ -136,8 +137,7 @@ docker run --rm -it -p 5173:80 alibaba:latest
 
 - [ ] improve `README.md` and add `Contribute`, `Folder Structure` sections
 - [ ] [Use icons from any icon sets with classes](https://github.com/antfu/)
-- [ ] SSG Support - [vite-ssg](https://github.com/antfu/vite-ssg)
-  - [ ] [`vite-ssg-sitemap`](https://github.com/jbaubree/vite-ssg-sitemap) - Sitemap generator
+- [ ] [`vite-ssg-sitemap`](https://github.com/jbaubree/vite-ssg-sitemap) - Sitemap generator
```
Comment (hossein13m) [inline README.md:140]: wouldn't it be nicer to check todo list items using `[x]` syntax instead of drawing a line over them?

**Review by hossein13m — APPROVED** [review]
seems OK but there are conflicts that needs to be resolved

File: README.md (hunk 1 above)
Comment (mohamadrezahedayati) [reply README.md:140]: The plugins used in the project come first. It is better for the user to know what is in the repo.
@hossein13m

**Review by mHooshdar — APPROVED** [review]

**Pull request merged**